apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts

```diff
@@ -288,7 +290,14 @@ export class ReportFiltersComponent implements OnInit, OnChanges, OnDestroy {
             if (changes?.stockReportRequest?.currentValue?.warehouseUniqueNames?.length) {
                 this.selectedWarehouse = changes?.stockReportRequest?.currentValue?.warehouseUniqueNames;
                 this.stockReportRequest.warehouseUniqueNames = this.selectedWarehouse;
-                this.balanceStockReportRequest.warehouseUniqueNames = this.selectedWarehouse;
+            }
+            if (changes?.stockReportRequestExport?.currentValue?.branchUniqueNames?.length) {
+                this.selectedBranch = changes?.stockReportRequestExport?.currentValue?.branchUniqueNames;
+                this.stockReportRequestExport.branchUniqueNames = this.selectedBranch;
```

Comment:
use stockReportRequest if interface is same and assign value into your interface from this interface.

Reply:
done